accounts/abi/bind/base.go

```diff
@@ -35,6 +36,8 @@ type SignerFn func(common.Address, *types.Transaction) (*types.Transaction, erro
 // CallOpts is the collection of options to fine tune a contract call request.
 type CallOpts struct {
 	Pending bool // Whether to operate on the pending state or the last known one
+
+	Ctx context.Context // unlike usual practice, this context can be left uninitialized (nil)
```

Comment:
Please rename this field to Context. I'd like to keep the API consistent. Also please change the comment to `Network context to support cancellation and timeouts (nil = no timeout)`.